src/main/java/com/chromediopside/controller/ProfileController.java

```diff
@@ -0,0 +1,34 @@
+package com.chromediopside.controller;
+
+import com.chromediopside.service.ErrorService;
+import com.chromediopside.service.ProfileService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class ProfileController {
+
+  private ProfileService profileService;
+  private ErrorService errorService;
+
+  @Autowired
+  public ProfileController(ProfileService profileService,
+      ErrorService errorService) {
+    this.profileService = profileService;
+    this.errorService = errorService;
+  }
+
+  @ExceptionHandler(Exception.class)
+  public ResponseEntity<?> exception(Exception ex) {
+    return errorService.getUnauthorizedResponseEntity();
+  }
+
+  @RequestMapping("/profile")
+  public ResponseEntity<?> getProfile(@RequestHeader(name = "X-GiTinder-token") String accessToken) throws Exception {
```

Comment:
Naming: we'll get an `appToken` here in the header, not an `accessToken`. Of course it works with the `accessToken` name as well, but it would be less confusing if we use the proper name here.